src/main/java/org/aksw/simba/squirrel/analyzer/impl/GuessMimeType.java

```diff
@@ -0,0 +1,51 @@
+package org.aksw.simba.squirrel.analyzer.impl;
+
+import java.io.File;
+import java.io.FileInputStream;
+import org.aksw.simba.squirrel.analyzer.Guess;
+import org.apache.tika.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GuessMimeType implements Guess {
+	private static final Logger LOGGER = LoggerFactory.getLogger(RDFAnalyzer.class);
+
+	public String guess(File data) {
+		String mimeType = null;
+		try {
+			FileInputStream inputStream = new FileInputStream(data.getAbsolutePath());
+			String fileContent = IOUtils.toString(inputStream);
+			if (fileContent.toString().isEmpty()) {
+				mimeType=null;
+			} else if (fileContent.toString().contains("<?xml") && fileContent.toString().contains("<rdf:RDF")) {
+				mimeType = "RDF/XML";

```

Comment:
Please make sure that you are using the official mimetypes for the different languages, e.g., `application/rdf+xml`. See http://www.iana.org/assignments/media-types/media-types.xhtml for the list of available mime types.

Please check whether jena is understanding these mime types correctly.